Monika After Story/game/script-topics.rpy

```diff
@@ -14195,3 +14195,28 @@ label monika_translating_poetry:
         m 5ekbsa "It would be nice if you could read some of them for me sometime..."
         m 5rkbsu "You'd have to translate them for me first, though~"
     return
+init 5 python:
+    addEvent(
+        Event(
+            persistent.event_database,
+            eventlabel="monika_diving",
+            category=["nature"],
+            prompt="Diving",
+            random=True
+        )
+     )
+label monika_diving:
+    m 3eub "[player], have you heard about diving?"
+    m 1eub "this is scuba diving with special equipment."
+    m 3wub "It's very exciting. Just imagine you are diving."
+    m 3eub "There can see a wonderful underwater world full of different marine life."
+    m 7eub "You can see: schools of fish, coral reefs, and sea greens."
+    m 4eub "In addition to predatory fish, sharks can be seen there."
+    m 2eud "As a rule, they are not in special places for diving."
+    m 7eud "And if there is, then they do not pose a threat."
+    m 2euc "Although the chance of shark attack is very small."
+    m 3esb "But if you meet a shark, here is one rule for you."
+    m 3esb "Do not make sudden movements."
```

Comment:
Instead of saying not to make any sudden movements, I think it would better explain it if she said to stay calm, since that also covers not making sudden movements, and it's probably another largely important rule.